core/entrypoint/docker-entrypoint.sh

```diff
@@ -1,12 +1,20 @@
 #!/bin/bash
 
 set -e
+trap 'echo "docker-entrypoint.sh : Error occurred on line $LINENO, exiting."; exit 1;' ERR
 
+# Ensure that variables are set
 if [ -z "$CHORUS_USER" ]; then
     echo "CHORUS_USER is not set. Defaulting to 'chorus'"
     export CHORUS_USER="chorus"
 fi
 
+if [ -z "$CARD" ]; then
+    echo "CARD is not set. Defaulting to 'none'"
+    CARD="none"
+fi
```

Comment:
it's an expensive way of doing this, but it's quite explicit, so good probably.

Reply:
Yep, don't really know which way things are going to evolve with the different apps and cases to deal with. Once this solidifies a little i am all for bringing the machete out and simplifying